src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -413,7 +413,11 @@ public enum CassandraRelevantProperties
     PARALLEL_INDEX_READ_NUM_THREADS("cassandra.index_read.parallel_thread_num"),
 
     // Allows skipping advising the OS to free cached pages associated commitlog flushing
-    COMMITLOG_SKIP_FILE_ADVICE("cassandra.commitlog.skip_file_advice");
+    COMMITLOG_SKIP_FILE_ADVICE("cassandra.commitlog.skip_file_advice"),
+
+    // Changes the semantic of the "THREE" consistency level to mean "all but one"
+    // i.e. that all replicas except for at most one in the cluster (across all DCs) must accept the write for it to be successful.
+    THREE_MEANS_ALL_BUT_ONE("dse.consistency_level.three_means_all_but_one", "false");
```

Comment:
I have used the name `dse.consistency_level.three_means_all_but_one` to ensure complete backwards compatibility but I am open to other suggestions. I admit it's a bit odd to start the name of it with `dse` while all of the other ones start with `cassandra`.